### PR TITLE
Add Unity package for AppLovin 4.8.4

### PR DESCRIPTION
**Pull request by chauduyphanvu1**

### CODE REVIEW

**Comment by PoojaChirp** [comment]
LGTM

File: unity-sample-app/Assets/MoPub/Editor/Support/MoPub-AdColony-Adapters/Changelog.md [hunk 1]
```diff
@@ -1,4 +1,8 @@
-## Changelog
+## Changelog
```
Comment (rsatrasala) [inline unity-sample-app/Assets/MoPub/Editor/Support/MoPub-AdColony-Adapters/Changelog.md]: Why is AdColony changing for AppLovin?
Also the version is jumping from 3.3.0.0 to 3.3.0.3

Comment (rsatrasala) [reply unity-sample-app/Assets/MoPub/Editor/Support/MoPub-AdColony-Adapters/Changelog.md]: Looks like this is a combined build for all changes in the sprint. Can you confirm?

Comment (chauduyphanvu1) [reply unity-sample-app/Assets/MoPub/Editor/Support/MoPub-AdColony-Adapters/Changelog.md]: https://github.com/mopub/mopub-ios-mediation/blob/master/AdColony/Changelog.md. The version change indicated a change in the podspec (to correctly pin a specific SDK version). Other than that there was no specific code change for AdColony in this release.

File: unity-sample-app/Assets/MoPub/Editor/Support/MoPub-Chartboost-Adapters/Changelog.md [hunk 2]
```diff
@@ -1,4 +1,8 @@
-## Changelog
+## Changelog
+  * 7.1.2.1
+    * This version of the adapters has been certified with Chartboost 7.1.2.
+    * Podspec version bumped in order to pin the network SDK version.
```
Comment (rsatrasala) [inline unity-sample-app/Assets/MoPub/Editor/Support/MoPub-Chartboost-Adapters/Changelog.md]: This log is not necessary 
"Podspec version bumped in order to pin the network SDK version."  <- This is the result of the change and not the cause. We only list causes.

File: unity-sample-app/Assets/MoPub/Editor/Support/MoPub-AdColony-Adapters/Changelog.md (hunk 1 above)
Comment (chauduyphanvu1) [reply unity-sample-app/Assets/MoPub/Editor/Support/MoPub-AdColony-Adapters/Changelog.md]: And yes, it's a combined build. Whatever changed after the previous release from the submoduled repos got pulled again to make new packages for this release.

**Comment by chauduyphanvu1** [comment]
Thanks, @rsatrasala. Comments have been addressed offline. Merging this PR.

**Pull request merged**